src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -567,7 +567,17 @@ type VehicleRentalStation implements Node & PlaceInterface {
     the rental station, even if the vehicle racks don't have any spaces available.
     See field `allowDropoffNow` to know if is currently possible to return a vehicle.
     """
-    spacesAvailable: Int
+    spacesAvailable: Int @deprecated(reason: "Use `availableSpaces` instead, which also contains the space vehicle types")
+
+    """
+    Number of vehicles currently available on the rental station, grouped by vehicle type.
+    """
+    availableVehicles: RentalVehicleEntityCounts!
+
+    """
+    Number of free spaces currently available on the rental station, grouped by vehicle type.
+    """
+    availableSpaces: RentalVehicleEntityCounts!
```

Comment:
```suggestion
    availableSpaces: RentalVehicleEntityCounts
```

Reply:
We decided that in order to be more flexible we will make these fields nullable.